src/components/forms/lock_actions/LockForm/components/VeBalForm/VeBalForm.vue

```diff
@@ -115,6 +115,11 @@ const lockType = computed(() => {
 function handleClosePreviewModal() {
   showPreviewModal.value = false;
 }
+
+function handleShowPreviewModal() {
+  if (!isValidLockEndDate.value) return;
```

Comment:
Shouldn't the button be disabled from `submissionDisabled` if this is the case?

Reply:
fixed